biosimulations/apps/dispatch-service/src/app/app.controller.ts

```diff
@@ -52,20 +55,22 @@ export class AppController {
       return { message: 'Unsupported simulator was provided!' };
     }
 
-    const omexPath = data.filepathOnDataStore;
-    const sbatchName = `${uuid()}.sbatch`;
+    const sbatchName = `${data.simulationId}.sbatch`;
     const sbatchPath = path.join(sbatchStorage, sbatchName);
 
     this.logger.log('SBatch path: ' + sbatchPath);
 
     // Generate SBATCH script
+    // TODO: Rename singularity images biosimulations_ to biosimulators_ on HPC and build new images according to /simulator from DB
```

Comment:
Should we use the API here to get the name of the docker image? The singularity images can then be called the same

Reply:
Yes, we can use the same name as docker images. I'll close this todo once I generate the singularity images on HPC and also I need to push the old Versions of COPASI into our packages.